.github/workflows/tag.yml

```diff
@@ -7,10 +7,10 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    env:
-      GITHUB_TOKEN: ${{ secrets.GOLANGCI_LINT_TOKEN }}
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
this will not work: for tags we need the full graph

Reply:
Removed that change.